services/catalog/src/simcore_service_catalog/services/registry.py

```diff
@@ -0,0 +1,78 @@
+""" NOTE that this not refer ONLY to the docker registry but rather access to all published services both
+    - as images in the docker-registry
+    - as code in function-services
+"""
+
```

Comment:
too many registered, registry for me here.
is it the registry in the end? or anything? and if you can get the service it means it is registered, be it in the registry or in this process...

Reply:
@sanderegg  I agree but I ran out of  names... I will try to review that again. IN the meantime this is the idea

Services can be registered in two ways:
1. pushing an image w/ labels+tags in the docker registry (e.g. Sleeper)
2. defining a "function-service" in the code (e.g. FilePicker, etc)

Reply:
check new explanation and name